daq/report.py

```diff
@@ -149,7 +153,7 @@ def finalize(self):
         for extension in ['.md', '.pdf', '.json']:
             if self._alt_path:
                 shutil.copyfile(self._report_prefix + extension, self._alt_prefix + extension)
-            report_paths.update({'report_' + extension: self._report_prefix + extension})
+            report_paths.update({'report' + extension: self._report_prefix + extension})
```

Comment:
The front end (main.js under handleFileLinks) uses "_path" in attribute (e.g. report_path) to determine if it needs to generate a link for file download.